src/System.Diagnostics.TraceSource/tests/TraceClassTests.cs

```diff
@@ -341,9 +351,42 @@ public void TraceTest01()
         }
 
         [Fact]
```

Comment:
> If we really need to know that we are on desktop we should be looking at the target framework from AppContext.TargetFrameworkName which has a much better defined contract for this and can even be parsed by TargetFramework type. FrameworkDescription is very loosely defined and is subject to change.

Why do we have FrameworkDescription then?  Or why doesn't FrameworkDescription just return AppContext.TargetFrameworkName?